src/pykeen/training/lcwa.py

```diff
@@ -94,10 +91,7 @@ def _mr_loss_helper(
         # First filter the predictions for true labels and then repeat them based on the repeat vector
         positive_scores = predictions[labels == 1][repeat_true_labels]
 
-        return self.model.compute_mr_loss(
-            positive_scores=positive_scores,
-            negative_scores=negative_scores,
-        )
+        return self.model.compute_mr_loss(positive_scores, negative_scores)
```

Comment:
The only difference here is not passing the parameters by keyword, but by position?

Reply:
correct. This will be necessary later to better unify the computation of loss functions (since they all take 2 arguments)

Reply:
There are 3 different layers before this gets called, and this is the first step in cleanup